spigot/core/src/main/kotlin/com/mineinabyss/geary/minecraft/events/Events.kt

```diff
@@ -25,6 +26,8 @@ public object EventComponentSerializer : FlatSerializer<Events, Map<String, List
 )
 
 public fun Event.event(entity: GearyEntity?, name: String) {
+    if (entity?.has<EventsDisabledComponent>() == true) return
```

Comment:
Depends on how do we want to work with disabled events.
I guess it's better to let this thing be iterative.